mobile_back_tflite/cpp/backend_tflite/tflite_c.cc

```diff
@@ -205,7 +205,7 @@ mlperf_backend_ptr_t mlperf_backend_create(
     //   if it is not specified in settings.
     // If we don't use batching, we will use shards_num=1,
     //   which is the default value.
-    backend_data->shards_num = 2;
+    backend_data->shards_num = 4;
```

Comment:
The app crashed when running Image Classification (offline) on my test Android phone (TFLite backend).
It looks like an Out Of Memory issue, caused by this change. Setting the shards_num back to 2 resolves the crash.